index.js

```diff
@@ -96,7 +96,13 @@ function serialize(name, val, options) {
 
   if (opt.domain) pairs.push('Domain=' + opt.domain);
   if (opt.path) pairs.push('Path=' + opt.path);
-  if (opt.expires) pairs.push('Expires=' + opt.expires.toUTCString());
+  if (opt.expires) 
+  {
+    var now = new Date(opt.expires);
+    if (now + '' === 'Invalid Date') throw new Error('expires should be a valid datestring, unix timestamp or Date object');
+    var utc = new Date(now.getTime() - now.getTimezoneOffset() * 60 * 1000);
```

Comment:
I have no idea what you're trying to accomplish here, but it produces the wrong date/time:
```js
var str = '03 08 2015 07:10:00 -0400';

console.log(new Date(str).toISOString()); // correct time: 2015-03-08T11:10:00.000Z

var now = new Date(str);
var utc = new Date(now.getTime() - now.getTimezoneOffset() * 60 * 1000);

console.log(utc.toISOString()); // wrong time: 2015-03-08T07:10:00.000Z
```